File: tests/transformers/csharp/WebSocketServiceBase.cs
using System;
using System.Collections.Generic;
using System.Reactive.Subjects;

namespace UniRpc.WebApplication
{
    public abstract class WebSocketServiceBase
    {
        public string __name { get; set; }
        public bool __isGeneric { get; set; }
        public List<string> __genericArguments { get; set; }
        public Subject<BaseMessage> __outgoing { get; set; }
        public string __user { get; set; }
        public string __group { get; set; }
        public virtual BaseMessage __invoke(BaseMessage message)
        {
            throw new NotImplementedException();
        }
    }
}

File: pywebsocket/UniRpc/WebsocketServiceBase.py
from typing import List, Coroutine
from abc import abstractmethod
from UniRpc.BaseMessage import BaseMessage

class WebsocketServiceBase:
    __reflection: str
    __isGeneric: bool
    __genericArguments: List[str]
    @abstractmethod
    def __outgoing(message: BaseMessage) -> None:
        pass
    __user: str
    __group: str
    @abstractmethod
    def __invoke(message: BaseMessage) -> Coroutine[BaseMessage]:
        pass